src/Injector.php

```diff
@@ -83,6 +90,7 @@ public function getContainer() : ContainerInterface
      * Returns the class name for the requested type
      *
      * @param string $type
+     * @return string
```

Comment:
This is useless, we decided not to duplicated the same information in PHPDocs if these are defined in method declaration, unless it contains any extra information (like description with something important).

Reply:
Right just saw this as well when commenting my intentions. You guys are fast :+1: